app/views/TimeoutView.scala.html

```diff
@@ -0,0 +1,37 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import config.AppConfig
+
+@this(
+    layout: Layout,
+    link: components.link,
+    button: components.button,
+    details: GovukDetails
+)
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@layout(pageTitle = messages("timeout.title"), isTimeout = true) {
+
+    <div class="govuk-grid">
+        <div class="govuk-grid-column-two-thirds">
```

Comment:
This would cause illogical markup - see screenshot:  
![image](https://user-images.githubusercontent.com/53828896/155553452-e15894d6-a0b4-4c8a-a3b9-b00c7e902c12.png)

You can use the isPageFullWidth boolean parameter in the layout to set it to two-thirds
isPageFullWidth = false

Reply:
Also it causes the text to sit very slightly off margin:
![image](https://user-images.githubusercontent.com/53828896/155554000-0201df6e-85f2-45a7-9443-b6e756047a9c.png)